server/tests/transactions.go

```diff
@@ -609,7 +609,6 @@ func (r *TransactionsRepository) readRow(ctx context.Context, row scanner, augme
 
 func removeNonAugmentedFields(t *Transaction) {
 	t.CreatedAt = nil
-	t.Version = nil
```

Comment:
We use that in the list endpoint, so I removed this from the augmented version

Reply:
this will also impact on the yaml file generation. why do you need the version on the transaction list? and if so, it would be easier to just request the augmented list